src/software/convert/main_convertRAW.cpp

```diff
@@ -33,6 +33,7 @@ int main(int argc, char** argv)
   std::vector<std::string> imagePaths;
   std::string outputFolder;
   std::string outImageFileTypeName = image::EImageFileType_enumToString(image::EImageFileType::EXR);
+  int nbImages = 0;
```

Comment:
Would be better to move it after the command line parsing to not confuse with the input parameters.